tests/rptest/tests/log_segment_ms_test.py

```diff
@@ -38,7 +38,10 @@ class SegmentMsTest(RedpandaTest):
     def __init__(self, test_context):
         super().__init__(test_context=test_context,
                          num_brokers=1,
-                         extra_rp_conf={"enable_leader_balancer": False})
+                         extra_rp_conf={
+                             "enable_leader_balancer": False,
+                             "log_segment_ms_min": 60000,
```

Comment:
nit: this would be better close to where it is expected to be used, ie ~ line 112
```
        """
        Tests under several conditions that a segment will not roll,
        either because there is no config or the topic property is set to disable.
        The topic is created with the configuration or an alter config is issued to configure it
        """

        num_messages = messages_to_generate_segments(TEST_NUM_SEGMENTS,
                                                     SERVER_SEGMENT_MS)

        start_count, stop_count = self.generate_workload(
-            server_cfg, topic_cfg, use_alter_cfg, num_messages)
+            server_cfg,
+            topic_cfg,
+            use_alter_cfg,
+            num_messages,
+            extra_cluster_cfg={
+                'log_segment_ms_min': SERVER_SEGMENT_MS,
+            })
```

Reply:
I went with this because log_segment_ms_min was meant as a way to let us write tests that do not require to wait too much. before this change the default was fine, but now setting log_segment_ms_min=60s it's important.

There are other tests that set in the constructor a property as a way to ensure a quick test

Reply:
Doesn’t my proposed change achieve the same? It just does it in a different place and using a constant that is set to the same value and, as I understand, needs to match the server config.

Reply:
Ah i aee what you mean, since just one or two tests actually need this property. I'll change it